### PR TITLE
Upgrade galaxy_ng to 4.2.1

### PR DESCRIPTION
**Pull request by bmbouter**

### CODE REVIEW

**Review by mdellweg — APPROVED** [review]
Looks reasonable. Can we make this kind of PR be part of galaxy_ng's release process?

File: pulp_galaxy_ng/Containerfile [hunk 1]
```diff
@@ -2,8 +2,8 @@ FROM pulp/pulp-ci:latest
 
 ARG PULPCORE_VERSION=">=3.7,<3.9"
 ARG PULP_CONTAINER_VERSION="==2.1.0"
-ARG PULP_ANSIBLE_VERSION="==0.5.0"
-ARG GALAXY_NG_VERSION="==4.2.0rc2"
+ARG PULP_ANSIBLE_VERSION="==0.5.5"
```
Comment (fao89) [inline pulp_galaxy_ng/Containerfile:5]: it will break ansible-galaxy CI, they need `0.5.4`

Comment (bmbouter) [reply pulp_galaxy_ng/Containerfile:5]: They on chat told me 0.5.5?

Comment (fao89) [reply pulp_galaxy_ng/Containerfile:5]: this is about the query issue, galaxy needs `0.5.5` but ansible-galaxy needs `0.5.4`

Comment (newswangerd) [reply pulp_galaxy_ng/Containerfile:5]: 4.2.1 is currently running 0.5.5. 4.2.2 will likely run 0.5.6 (assuming there aren't any new 0.5x releases that we need before 4.2.2 goes out)

Comment (fao89) [reply pulp_galaxy_ng/Containerfile:5]: @jborean93 does your CI have specific tests for `Collection.updated_at`? Let us know if this change is breaking your CI
(sorry about that, we plan to add the query back ASAP)

Comment (jborean93) [reply pulp_galaxy_ng/Containerfile:5]: Sorry I've been distracted with a few other things recently. Our CI pins our `pulp_galaxy_ng` container to a specific hash so any updates won't affect us until we actually upgrade.

**Pull request merged**